README.md

```diff
@@ -97,6 +107,17 @@ To load mock interventions, run `python scripts/load_interventions.py --authz 'T
 
 To generate black spot and load forecast training inputs, run `python scripts/generate_training_input.py /path/to/roads.shp /path/to/records.csv`.
 
+### Costs
+
+You can't request records with associated costs successfully until you configure some costs.
```

Comment:
Wait, are you saying that all record requests fail if this isn't configured? We should probably make an issue for that if so, it's not supposed to impact the rest of the app.

Reply:
Not _all_ records requests (like, you can hit `/api/records/` successfully), but you can't hit `/api/records/costs/`, which is required for a lot of the app. I _think_ you can view the records list but I don't remember.